tests/integration/server/proposals_api_test.py

```diff
@@ -13,7 +13,115 @@
 # limitations under the License.
 # -----------------------------------------------------------------------------
 """Integration tests for proposals APIs"""
+import requests
 
-# TODO: Add tests for the proposals API
-# TODO: Add a test to check that users from the manager chain are in the
-#      approvers list.
+from tests.utilities import (
+    add_role_member,
+    create_test_user,
```

Comment:
Minor nit pick, but if you could put the imports in alphabetical order that would be great!